client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -26,83 +26,76 @@ import { FEATURE_SIMPLE_PAYMENTS } from 'lib/plans/constants';
 
 class SimplePaymentsView extends Component {
 	render() {
-		const {
-			translate,
-			productId,
-			product,
-			siteId,
-			sitePlan,
-			planHasSimplePaymentsFeature,
-		} = this.props;
-
-		if ( ! product ) {
-			return <QuerySimplePayments siteId={ siteId } productId={ productId } />;
-		}
+		const { productId, product, siteId } = this.props;
+
+		return (
+			<div className="wpview-content wpview-type-simple-payments">
+				<QuerySimplePayments siteId={ siteId } productId={ productId } />
+				<QuerySitePlans siteId={ siteId } />
+				{ product && <QueryMedia siteId={ siteId } mediaId={ product.featuredImageId } /> }
+				{ this.renderContent() }
+			</div>
+		);
+	}
+
+	renderContent() {
+		const { translate, product, productImage, planHasSimplePaymentsFeature, sitePlan } = this.props;
 
-		if ( ! sitePlan ) {
-			return <QuerySitePlans siteId={ siteId } />;
+		if ( ! product || ! sitePlan ) {
+			return;
```

Comment:
If it's not too much trouble, could we return a placeholder in this case? On a clean cache, there is a bit of pop-in